apps/emqx_ft/src/emqx_ft_schema.erl

```diff
@@ -275,11 +292,19 @@ validator(filename) ->
     ];
 validator(backend) ->
     fun(Config) ->
-        case maps:keys(Config) of
-            [_Type] ->
+        Enabled = maps:filter(
+            fun(_, Backend) ->
+                maps:get(enable, Backend, false) or maps:get(<<"enable">>, Backend, false)
```

Comment:
Tests were failing in `emqx_ft_schema:translate/1` calls. Since [9159f4a](https://github.com/emqx/emqx/pull/10974/commits/9159f4a3835288ea500a45a830f5ef8df7dd4e65) shouldn't be needed anymore.